sim/network.go

```diff
@@ -34,7 +32,13 @@ const (
 
 const _ = TraceAll // Suppress unused constant warning.
 
+type signingBackend interface {
+}
```

Comment:
```suggestion
type signingBackend any
```

Reply:
This needs to get deleted as it is a dev remnant.